docs/Merge-MarkdownHelp.md

```diff
@@ -14,7 +14,7 @@ Merge multiple markdown versions of the same cmdlet into a single markdown file.
 
 ```
 Merge-MarkdownHelp [-Path] <String[]> [-OutputPath] <String> [-Encoding <Encoding>] [-ExplicitApplicableIfAll]
- [-Force] [[-MergeMarker] <String>] [<CommonParameters>]
```

Comment:
I think for the merge case we don't need this parameter. Merge operates purely on the markdown files, so assuming that original markdown files include them, we will get them included automatically.